WordPress/src/main/java/org/wordpress/android/ui/mysite/menu/MenuActivity.kt

```diff
@@ -0,0 +1,320 @@
+        LazyColumn(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(16.dp),
+            verticalArrangement = Arrangement.spacedBy(8.dp),
+        ) {
+            items(uiState.items) { viewState ->
+                when (viewState) {
+                    is MySiteCardAndItem.Item.ListItem -> MySiteListItem(viewState)
+                    is MySiteCardAndItem.Item.CategoryHeaderItem -> MySiteListItemHeader(viewState)
+                    is MySiteCardAndItem.Item.CategoryEmptyHeaderItem -> MySiteListItemEmptyHeader()
+                    else -> {
+                    }
+                }
+            }
+        }
+    }
+
+    @Composable
+    fun MySiteListItemHeader(headerItem: MySiteCardAndItem.Item.CategoryHeaderItem) {
+        val title = when (headerItem.title) {
+            is UiString.UiStringRes -> stringResource(id = headerItem.title.stringRes)
+            is UiString.UiStringText -> headerItem.title.text.toString()
+            is UiString.UiStringPluralRes -> TODO()
+            is UiString.UiStringResWithParams -> TODO()
+        }
+        Text(
+            modifier = Modifier.padding(8.dp),
+            fontSize = 14.sp,
+            fontWeight = FontWeight.Medium,
+            color = MaterialTheme.colors.onSurface.copy(alpha = ContentAlpha.medium),
+            text = title)
+    }
+
+    @Composable
+    fun MySiteListItemEmptyHeader() {
+        Spacer(modifier = Modifier.height(4.dp))
+    }
+
+    @Composable
+    fun MySiteListItem(item: MySiteCardAndItem.Item.ListItem) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentSize()
+                .padding(vertical = 8.dp, horizontal = 8.dp)
+                .clickable { item.onClick.click() },
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Image(
+                painter = painterResource(id = item.primaryIcon),
+                contentDescription = null, // Add appropriate content description
+                contentScale = ContentScale.Fit,
+                modifier = Modifier
+                    .size(24.dp)
+                    .padding(end = 8.dp),
+                colorFilter =
+                    if (item.disablePrimaryIconTint) null else ColorFilter.tint(MaterialTheme.colors.onSurface)
+            )
+
+            Text(
+                text = stringResource(id = (item.primaryText as UiString.UiStringRes).stringRes),
+                fontSize = 16.sp,
+                fontWeight = FontWeight.Medium,
+                color = MaterialTheme.colors.onSurface.copy(alpha = ContentAlpha.medium),
+                modifier = Modifier
+                    .weight(1f)
+                    .padding(end = 8.dp),
+            )
+            Spacer(modifier = Modifier.height(4.dp))
+
+            // todo: eventually we can take uiStringRes out of the state, but for now it's shared, so leave it
```

Comment:
Hey @zwarm 
💡  AFAIK only plan item is using UiStringText and @ravishanker confirmed that we can safely remove plans feature from the app since they are planning a revamp for plans in site creation.
We can take up this change in a seperate PR but just letting you know.